packages/@coorpacademy-components/src/molecule/base-modal/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../../atom/icon';
+import style from './style.css';
+
+const BaseModal = props => {
+  const {title, description, headerIconName, children, isOpen, footer, onClose} = props;
+
+  if (!isOpen || !title || !children) return null;
+
+  function handleOnClose(e) {
+    e?.stopPropagation();
+    onClose();
+  }
+
+  return (
+    <div className={style.modalWrapper}>
+      <div className={style.modal}>
+        <header className={style.header}>
+          {headerIconName ? (
+            <div className={style.headerIcon}>
+              <Icon
+                iconName={headerIconName}
+                backgroundColor="#DDD1FF"
```

Comment:
would be interesting to have it a props:
`icon?: {name, color?}`
(color could have a default)